apps/console/src/protected-app.tsx

```diff
@@ -151,18 +154,23 @@ export const ProtectedApp: FunctionComponent<AppPropsInterface> = (): ReactEleme
                 .get(Config.getServiceResourceEndpoints().wellKnown)
                 .then((response: AxiosResponse) => {
                     // Use token endpoint to extract the host url.
-                    const splitted: string[] = response.data.token_endpoint?.split("/") ?? [];
+                    const splitted: string[] = response?.data?.token_endpoint?.split("/") ?? [];
                     const serverHost: string = splitted.slice(0, -2).join("/");
-                    
+
                     window[ "AppUtils" ].updateCustomServerHost(serverHost);
+                })
+                .catch((error) => {
+                    // In case of failure customServerHost is set to the serverHost
+                    window[ "AppUtils" ].updateCustomServerHost(Config.getDeploymentConfig().serverHost);
+                    
+                    throw error;
+                })
+                .finally(() => {
```

Comment:
The catch block was added for safety since there are split, slice functions. We usually do not explicitly catch errors of dispatch, setRuntimeConfig functions' errors.